WeightProducer/python/MCSampleValues.py

```diff
@@ -251,669 +247,501 @@ class MCSampleValuesHelper():
+                 XS_13TeV=746.2, XSSource_13TeV="XSDB (NLO)",
             ),
         },
-        "WJetsToLNu_HT-2500ToInf" : {
-            "CrossSection" : XSValues(
-                XS_2016=0.03209, XSSource_2016="XSDB (LO)",
-                XS_2017=0.008001, XSSource_2017="XSDB (LO)",
-                XS_2018=0.008060, XSSource_2018="GenXSecAnalyzer",
+        "TTToSemiLeptonic_mtop171p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=706.1, XSSource_13TeV="XSDB (NLO)",
             ),
-            "kFactor" : kFactorValues(
-                kFactor_2016=1.224, kFactorSource_2016="XSDB NNLO/LO=61526.7/50260",
-                kFactor_2017=1.162, kFactorSource_2017="XSDB NNLO/LO=61526.7/52940",
-                kFactor_2018=1.164, kFactorSource_2018="XSDB NNLO/LO=61526.7/52850",
+        },
+        "TTToSemiLeptonic_mtop173p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=668.6, XSSource_13TeV="XSDB (NLO)",
             ),
-            "Correction" : CorrValues(
-                Corr_2017=4.200, CorrSource_2017="https://twiki.cern.ch/twiki/bin/viewauth/CMS/MCKnownIssues#WJetsToLNu_HT_and_DYJets_HT_LO_M",
-                Corr_2018=4.200, CorrSource_2018="Same as 2017",
+        },
+        "TTToSemiLeptonic_mtop175p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=633.4, XSSource_13TeV="XSDB (NLO)",
             ),
         },
-        "WJetsToLNu" : {
-            "CrossSection" : XSValues(
-                XS_13TeV=61526.7, XSSource_13TeV="https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#W_jets NNLO (60430.0 @ NLO)",
-                XS_2016=50260.0, XSSource_2016="XSDB (LO)",
-                XS_2017=52940.0, XSSource_2017="XSDB (LO)",
-                XS_2018=52850.0, XSSource_2018="XSDB (LO)",
+        "TTToSemiLeptonic_mtop178p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=584.6, XSSource_13TeV="XSDB (NLO)",
             ),
         },
-        "WJetsToQQ_HT400to600_qc19_3j" : {
+        "TTGamma_SingleLept" : {
             "CrossSection" : XSValues(
-                XS_2017=315.2, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=314.6, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=5.085, XSSource_13TeV="XSDB (LO)",
             ),
         },
-        "WJetsToQQ_HT600to800_qc19_3j" : {
+        "TTGamma_Hadronic" : {
             "CrossSection" : XSValues(
-                XS_2017=68.61, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=68.58, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=4.178, XSSource_13TeV="XSDB (LO)",
             ),
         },
-        "WJetsToQQ_HT-800toInf_qc19_3j" : {
+        "TTGamma_Dilept" : {
             "CrossSection" : XSValues(
-                XS_2017=34.71, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=34.74, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=1.502, XSSource_13TeV="XSDB (LO)",
```

Comment:
Are the `TTGamma_SingleLept`, `TTGamma_Hadronic`, and `TTGamma_Dilep` cross sections the same for all years? They used to be different for each year. Also, are the samples `TTGamma_SingleLeptFromT` and `TTGamma_SingleLeptFromTbar` samples combined into the `TTGamma_SingleLept` sample? If so, why is it's cross section so much higher than the sum of the old ones.

Reply:
So far, I have not been able to find any evidence that cross sections will be different on a per-year basis. I do find [1], which discusses a uniform approach for MC generation for all three UL years, helping support the assumption that all three years are indeed treated equally.

From what I can tell, the top and antitop modes for single lepton are now just one.  As for the cross section discrepancy, another thing to investigate.

[1] https://cms-pdmv.gitbook.io/project/mccontact/info-for-mc-production-for-ultra-legacy-campaigns-2016-2017-2018

Reply:
I also don't see the value of 1.502 in the XSDB for the search `DAS=TTGamma_Dilept && DAS=TuneCP5`. Just thought I'd mention that.

Reply:
I guess this [2] is what I have been going off of for Dilept in particular.

[2] https://cms-gen-dev.cern.ch/xsdb/?columns=67108800&currentPage=0&pageSize=10&searchQuery=DAS%3DTTGamma_Dilept_TuneCP5_13TeV-madgraph-pythia8

Reply:
Okay, I see the value for TTGamma_Dilept in the XSDB now, but that still doesn't explain the XS change. Not unless the madgraph generation has changed.

Reply:
Hmm, when I specifically follow a Fall17 or Autumn18 TTGamma_Dilept sample _to_ XSDB [3] from DAS [4], it points to an entry with a cross section very close to the 1.5 value. For Summer16 samples with the TuneCUETPM8 tune, I do find the ~0.6 cross section. There are XSDB entries for Fall17 that have the smaller value reported and were part of a different chain, but otherwise have the same DAS name: compare top two rows [5].

The MadGraph version did change from 2.6.0 to 2.6.5.

[3] https://cmsweb.cern.ch/das/request?input=dataset%3D%2FTTGamma_Dilept_TuneCP5_13TeV-madgraph-pythia8%2FRunIIAutumn18MiniAOD-102X_upgrade2018_realistic_v15-v1%2FMINIAODSIM&instance=prod/global

[4] https://cms-gen-dev.cern.ch/xsdb/?searchQuery=DAS=TTGamma_Dilept_TuneCP5_13TeV-madgraph-pythia8

[5] https://cms-gen-dev.cern.ch/xsdb/?columns=67108863&currentPage=0&pageSize=10&searchQuery=DAS%3DTTGamma_Dilept_TuneCP5_